sonaranalyzer-dotnet/tests/SonarAnalyzer.UnitTest/TestCases/XmlExternalEntityShouldNotBeParsed_XmlDocument_CSharp9.cs

```diff
@@ -0,0 +1,17 @@
+﻿using System.Xml;
+using System.Xml.Resolvers;
+
+XmlDocument xmlDocument = new(); // Compliant - default constructor is safe in .Net 5
+
+XmlDocument doc1 = new() { XmlResolver = new XmlPreloadedResolver() }; // Compliant - FN
```

Comment:
to fix this, we should have a yellow _Implement_ value in the spreadsheet for XmlExternalEntityShouldNotBeParsed, right? Target typed new doesn't work in this case